--- extras/package/macosx/package.mak	2018-10-05 14:10:43.000000000 +0200
+++ extras/package/macosx/package.mak	2018-10-05 14:11:43.000000000 +0200
@@ -45,7 +45,7 @@
 	mkdir -p $(top_builddir)/tmp/modules/gui/macosx
 	cd "$(srcdir)/modules/gui/macosx/" && cp *.h *.m $(abs_top_builddir)/tmp/modules/gui/macosx/
 	cd $(top_builddir)/tmp/extras/package/macosx && \
-		xcodebuild -target vlc SYMROOT=../../../build DSTROOT=../../../build $(silentstd)
+		xcodebuild -UseNewBuildSystem=NO -target vlc SYMROOT=../../../build DSTROOT=../../../build $(silentstd)
 	cp -R $(top_builddir)/tmp/build/Default/VLC.bundle $@
 	mkdir -p $@/Contents/Frameworks && cp -R $(CONTRIB_DIR)/Growl.framework $@/Contents/Frameworks/
 	mkdir -p $@/Contents/MacOS/share/locale/
